RHEL6_7/services/openldap/check

```diff
@@ -55,22 +55,22 @@ If you are using the bdb or hdb openldap back ends, it is necessary to update
 the database.
 
 Back up the data:
-`# slapcat -l backup.ldif`
+# slapcat -l backup.ldif
```

Comment:
This line and similar lines below actually change behavior: in previous version, the code between the backticks would be actually executed.  It's just due to the fact that all instances of this are prefixed with hash sign, leading to "execution" of comments, ie. not doing anything and just returning empty string.

(Technically, this should have been in different commit.)

Reply:
Excuse my ignorance, but why would anyone execute a comment?

Reply:
That's my point; the code was never supposed to be executed.  We're just lucky that the code actually happens to be interpreted as comment.

Reply:
Just want to point out, that actually this is not comment, but text representing the command that the root should do. The idea was use the markdown notation I guess.